doc/releases/octopus.rst

```diff
@@ -1,3 +1,104 @@
+v15.2.3 Octopus
+===============
+
+This is the third bug-fix release of the Ceph Octopus stable release series.
+This release mainly is a workaround for a potential OSD corruption in v15.2.2.
+We advise users to upgrade to v15.2.3 directly. For users running v15.2.2 please
+execute the following::
+
+  ceph config set osd bluefs_preextend_wal_files false
+
+Changelog
+---------
+* bluestore: common/options.cc: disable bluefs_preextend_wal_files (`issue#45613 <http://tracker.ceph.com/issues/45613>`_, Neha Ojha)
```

Comment:
Could we change this line to

```
* bluestore: remove preextended WAL support
```

? That's the title of the commit message and it more accurately describes what the fix is doing.

Reply:
see the master PR https://github.com/ceph/ceph/pull/35497